code/game/objects/items/weapons/flamethrower.dm

```diff
@@ -167,12 +168,12 @@
 
 
 /obj/item/weapon/flamethrower/proc/ignite_turf(turf/target)
-	world << "Igniting turf [target.x] [target.y]"
 	//TODO: DEFERRED Consider checking to make sure tank pressure is high enough before doing this...
 	//Transfer 5% of current tank air contents to turf
 	var/datum/gas_mixture/air_transfer = ptank.air_contents.remove_ratio(0.02*(throw_amount/100))
-	//air_transfer.toxins = air_transfer.toxins * 5 // This is me not comprehending the air system. I realize this is retarded and I could probably make it work without fucking it up like this, but there you have it. -- TLE
+	air_transfer.multiply(gas_mult)
 	new/obj/effect/decal/cleanable/liquid_fuel/flamethrower_fuel(target,air_transfer.gas["plasma"],get_dir(loc,target))
```

Comment:
Move the multiplier here like this `air_transfer.gas["plasma"]*gas_mult`, no more actual gas fabrication.